rxtools/source/features/CTRDecryptor.c

```diff
@@ -8,13 +8,13 @@
 #include "hid.h"
 #include "ncch.h"
 #include "crypto.h"
+#include "stdio.h"
 
-#define BUFFER_ADDR ((volatile u8*)0x21000000)
+#define BUFFER_ADDR ((u8*)0x21000000)
```

Comment:
Be careful when using u8* not vu8* that contains volatile attribute. Someone compiler would like to cache value (in macro areas). volatile tells it this should never be cached.